### PR TITLE
Updated dataset description of DaNE

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**

### CODE REVIEW

File: datasets/dane/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ task_ids:
 - named-entity-recognition
 - part-of-speech-tagging
 paperswithcode_id: dane
+pretty_name: 'Danish Dependency Treebank (DaNE)
```
Comment (albertvillanova) [inline datasets/dane/README.md:21]: That leading single quote `'` is generating a YAML parsing error.

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Thanks for your contribution.

We are aware of the importance of metadata in order to find the datasets among all in our [Datasets Hub](https://huggingface.co/datasets).

This is why we are requiring some minimal information in the dataset card in order to pass our Continuous Integrations tests. In this case, the tests do not pass because there were some missing informations (that before were not required but now they are):
```
E           ValueError: The following issues have been found in the dataset cards:
E           README Validation:
E           The following issues were found for the README at `/home/circleci/datasets/datasets/dane/README.md`:
E           -	Expected some content in section `Data Instances` but it is empty.
E           -	Expected some content in section `Data Splits` but it is empty.
E           -	Expected some content in section `Citation Information` but it is empty.
``` 

That would be awesome if you could add this "required" information in the sections:
- Data Instances
- Data Splits
- Citation Information

If you would like to kindly add this information, please have a look at:
  - Our [Dataset Card Creation Guide](https://github.com/huggingface/datasets/blob/master/templates/README_guide.md),
  - and the corresponding section in [How to add one (or several) new datasets to 🤗 Datasets](https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card).

File: datasets/dane/README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ task_ids:
 - named-entity-recognition
 - part-of-speech-tagging
 paperswithcode_id: dane
+pretty_name: Danish Dependency Treebank (DaNE)
 ---
 
 # Dataset Card for [Dataset Name]
```
Comment (KennethEnevoldsen) [inline datasets/dane/README.md:24]:
```suggestion
# Dataset Card for DaNE
```

File: datasets/dane/README.md [hunk 3]
```diff
@@ -56,15 +57,17 @@ paperswithcode_id: dane
...
 
-[More Information Needed]
+Parts-of-speech tagging, dependency parsing and named entitity recognition.
 
 ### Languages
 
-[More Information Needed]
+Danish
 
 ## Dataset Structure
```
Comment (KennethEnevoldsen) [inline datasets/dane/README.md:73]:
```suggestion
### Data Splits


|                            | Tain   | Dev | Test |
| -----                      | ------ | ----- | ---- |
| # sentences            | 4383 | 564 | 565 |
| # tokens   | 80 378 | 10 322 | 10 023 |

```

File: datasets/dane/README.md [hunk 4]
```diff
@@ -154,4 +157,4 @@ Data Fields:
```
Comment (KennethEnevoldsen) [inline datasets/dane/README.md:157]:
```suggestion

### Citation Information

```
@inproceedings{hvingelby2020dane,
  title={DaNE: A named entity resource for danish},
  author={Hvingelby, Rasmus and Pauli, Amalie Brogaard and Barrett, Maria and Rosted, Christina and Lidegaard, Lasse Malm and S{\o}gaard, Anders},
  booktitle={Proceedings of the 12th Language Resources and Evaluation Conference},
  pages={4597--4604},
  year={2020}
}
```

**Review by KennethEnevoldsen — COMMENTED** [review]
Added a few of the metadata still missing the instance.

File: datasets/dane/README.md [hunk 5]
```diff
@@ -152,6 +162,8 @@ Data Fields:
 
 [More Information Needed]
 
+### Citation Information
+
```
Comment (KennethEnevoldsen) [inline datasets/dane/README.md:166]:
```suggestion

@inproceedings{hvingelby2020dane,
  title={DaNE: A named entity resource for danish},
  author={Hvingelby, Rasmus and Pauli, Amalie Brogaard and Barrett, Maria and Rosted, Christina and Lidegaard, Lasse Malm and S{\o}gaard, Anders},
  booktitle={Proceedings of the 12th Language Resources and Evaluation Conference},
  pages={4597--4604},
  year={2020}
}

```

**Review by KennethEnevoldsen — COMMENTED** [review]
Added citation inf.

**Comment by KennethEnevoldsen** [comment]
Thanks for finishing it @albertvillanova

**Pull request merged**